docs/plugins/events.md

```diff
@@ -0,0 +1,165 @@
+# Events
+
+_work in progres_
+
+Events plugin is used to listen to events emitted from:
+
+- various SDK plugins (i.e. VideoPlayer plugin, Router plugin)
+- the transport layer (for realtime changes in the Platform or Lifecycle)
+- custom App events broadcasted from the App code
+
+## Usage
+
+In order to use the Events plugin, import it from the Lightning SDK.
+
+```js
+import { Events } from '@lightningjs/sdk'
+```
+
+## Available methods
+
+### Listen
+
+Listen to events and execute a callback when they occur.
+
+Examples:
+
+1) Listen to a specific event on a specific Plugin.
+
+```js
+Events.listen('Lifecycle', 'pause', () => {
+  console.log('The App is going in a pause state')
+})
+
+Events.listen('Platform', 'device.hdcp', value => {
+  console.log('Device HDCP version changed to ' + value)
+})
+```
+
+When listening for a specific event (on a specific Plugin), the callback function is passed a `value` as its first and only argument.
+
+2) Listen to all events of a specific Plugin
+
+```js
+Events.listen('Lifecycle', (event, value) => {
+  if(event === 'init') {
+    // execute init functionality
+  }
+  if(event === 'close') {
+    // execute close functionality
+  }
+})
```

Comment:
I do like the way this looks w/ the listener functions declared right inside of the `listen()` call, but what if someone wants to reuse an existing function, e.g.:

```javascript
function myCustomListener(plugin, event, value) {
  if (plugin === 'Platform') {
    if (event === 'device.hdcp') {
      ...
    }
    else if (event === 'accessibility.closedCaptionsToggled') {
      ...
    }
  }
}

Events.listen('Platform', 'device.hdcp', myCustomListener)
Events.listen('Platform', 'accessibility.closedCaptionsToggled', myCustomListener)
```

in that case, you need all three variables passed to each version of `listen` to get it to work.

Listening to all events seems like it might not be the most efficient way around this.